docs-new/docs/graphql/core/api-reference/index.mdx

```diff
@@ -0,0 +1,111 @@
+---
+description: Hasura API reference
+keywords:
+  - hasura
+  - docs
+  - API
+  - API reference
+slug: index
+---
+
+# API Reference
+
+## Available APIs
+
+| API                            | Endpoint                                         | Access           |
+|--------------------------------|--------------------------------------------------|------------------|
+| GraphQL                        | [/v1/graphql](#graphql-api)                      | Permission rules |
+| Relay                          | [/v1beta1/relay](#relay-api)                     | Permission rules |
+| Legacy GraphQL                 | [/v1alpha1/graphql](#graphql-api)                | Permission rules |
+| Schema *(> v2.0)*              | [/v2/query](#schema-api)                         | Admin only       |
+| Metadata *(> v2.0)*            | [/v1/metadata](#metadata-api)                    | Admin only       |
+| Schema/Metadata *(deprecated)* | [/v1/query](#schema-metadata-api-deprecated)     | Admin only       |
```

Comment:
The original file seems to not have `-deprecated` suffixed in id. So, lets remove it here too

`#schema-metadata-api` should suffice

Reply:
Fixed